frontend/src/Commons/Forms/InputModules/Crops/CropsInfo.tsx

```diff
@@ -0,0 +1,272 @@
+/**
+ * @summary Crops Information Input Module
+ * @description A module with input fields to be used in a Form Module
+ * @author @Kcaparas
+ */
+
+import InputModuleInterface from 'src/Interface/InputModuleinterface';
+import { faSeedling } from '@fortawesome/free-solid-svg-icons';
+import InputModuleProps from 'src/Interface/InputModuleProps';
+import React, { useState } from 'react';
+import * as Yup from 'yup';
+import { Formik, Form } from 'formik';
+import ComponentText from '@Constants/ComponentText';
+import Button from '@Commons/Button/Button';
+import CropsInitialDetails from '@Constants/InitialCropsDetails';
+import FarmDetailsInterface from 'src/Interface/FarmDetailsInterface';
+import FieldDetailInterface from 'src/Interface/FieldDetailsInterface';
+import { SubmissionCropsInterface } from 'src/Interface/CropsDetailsInterface';
+import { CROPS_INFORMATION } from '@Constants/ModuleIDs';
+import CustomSelect from '@Commons/Input/Select/CustomSelect';
+import {
+  CropIDOptions,
+  PlantAgeOptions,
+  PlantsPerAcre,
+  DistanceBtwnPlants,
+  DistanceBtwnRows,
+  YesOrNo,
+  WherePruningsGo,
+} from '@Constants/CropOptions';
+import CustomField from '@Commons/Input/Field/CustomField';
+import CropsButtonGroup from './CropsButtonGroup';
+import {
+  StyledCropsSmallGroup,
+  StyledCropsLargeGroup,
+  StyledAddCancelButtonGroup,
+} from './CropsInfo.styles';
+import {
+  StyledFarmInfo,
+  StyledButtonGroupContainer,
+  StyledAreaContainer,
+} from '../../../FormStyles.styles';
```

Comment:
Maybe use `@Commons/...`
Is there a better directory for `FormStyles.styles` though?

Reply:
Commons I guess